playbooks/openshift-master/private/tasks/enable_bootstrap_config.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Setup the node group config maps
+  import_role:
+    name: openshift_node_group
+
+- name: Setup the node bootstrap auto approver
+  import_role:
+    name: openshift_bootstrap_autoapprover
+  when:
+  - openshift_master_bootstrap_auto_approve | default(False) | bool
```

Comment:
Should this default to True?  Trying to understand the process and if this is the default behavior.  My current install requires openshift_master_bootstrap_auto_approve=True in inventory.

Reply:
If you turn it off what happens?  I have successful installs without it, because the playbooks run CSR themselves.  If you're seeing failures it may be other inventory flags?

Reply:
If I don't use `openshift_master_bootstrap_auto_approve=True`, the compute nodes fail, but the playbook continues and processes the entire rest of the install and then reports a failure at the end.  Upon checking cluster status, everything is up as expected.


```
Failure summary:


  1. Hosts:    ec2-34-235-156-129.compute-1.amazonaws.com, ec2-54-227-73-138.compute-1.amazonaws.com
     Play:     Distribute bootstrap and start nodes
     Task:     openshift_node : fail
     Message:  Node start failed.
```

The task where the nodes actually fail is here:
```
TASK [openshift_node : Start and enable node for bootstrapping]
```
http://file.rdu.redhat.com/~rteague/temp/2018-04-10T15.18.40-deploy.log